keras/callbacks.py

```diff
@@ -939,7 +939,7 @@ def on_epoch_end(self, epoch, logs=None):
                 continue
             summary = tf.Summary()
             summary_value = summary.value.add()
-            summary_value.simple_value = value.item()
+            summary_value.simple_value = value if isinstance(value, float) else value.item()
```

Comment:
Prefer using `if isintance(...):` / etc: it makes lines shorter and more readable, and it will be extensible to more types in the future.